react-app/src/App.js

```
@@ -62,7 +65,7 @@ function App() {
         <ProtectedRoute path='/test' exact={true} >
         </ProtectedRoute>
         <Route>
-          Page Not Found
```

We could throw together a "Page Not Found" page so that users who end up at an incorrect page aren't confused why they're at the homepage again.